### ISSUE TITLE
[BUG] Loss scaling factor applied to labels instead of loss in _loss_fn_seg (inconsistent with paper description)

### ISSUE THREAD

**Issue by Pikauba** [MouseLand/cellpose#1339]
**Describe the bug**
According to the article ["Self-supervised learning for cell segmentation with SAM-Cellpose"](https://www.biorxiv.org/content/10.1101/2025.04.28.651001v1.full), the Cellpose segmentation loss is described as:  

> "The loss function was the Cellpose segmentation loss [3]: the mean squared error between the XY flows from the ground-truth segmentation and the predicted XY flows, scaled by a factor of five."  

However, in the current implementation of `_loss_fn_seg` (see [train.py#L48](https://github.com/MouseLand/cellpose/blob/0a3de7b6420470c1d1ccf33ba6f4bc821fe532cc/cellpose/train.py#L48)), the scaling factor of 5 is applied directly to the **labels** (`lbl[:, -2:]`) instead of the **loss**:  

```python
veci = 5. * lbl[:, -2:]
loss = criterion(y[:, -3:-1], veci)
```

This implementation forces the model to predict flows that are 5x larger than the true ground-truth flows, which differs from what is described in the paper (where the loss itself should be scaled by 5, not the label values).

This discrepancy can lead to unintended training dynamics and confusion when reproducing results described in the paper.

**To Reproduce**  
1. Inspect the `_loss_fn_seg` implementation in `cellpose/train.py`.  
2. Compare with the description in the SAM-Cellpose paper (Methods section, Loss Function).  
3. Observe that scaling is applied to the ground-truth flow labels rather than to the loss term.  

---

**Expected behavior**  
The scaling factor of 5 should be applied to the **loss value** (e.g., `loss = criterion(y[:, -3:-1], veci)`) instead of scaling the ground-truth labels.  
This would align with the description in the SAM-Cellpose paper and ensure the loss matches its intended definition.

**Comment by mrariden** [comment]
duplicate of #1299
